server.js

```diff
@@ -38,24 +42,33 @@ const formatTime = (timeInSeconds) => {
 };
 
 const welcomeHandler = (message) => {
-  bot.sendMessage(
-    message.from.id,
-    "You can Send / Forward anime screenshots to me. I can't get images from URLs, please send the image directly to me ;)"
-  );
+  bot.sendMessage(message.from.id, infoText);
 };
 
-const submitSearch = (buffer) =>
+// Expects image buffer by default
+const submitSearch = (imgData, kind) =>
   new Promise(async (resolve, reject) => {
-    const form = new FormData();
-    form.append("image", buffer, "blob");
     let response = {};
-    try {
-      response = await fetch(`https://trace.moe/api/search?token=${TRACE_MOE_TOKEN}`, {
-        body: form,
-        method: "POST",
-      });
-    } catch (error) {
-      reject(error);
+    if (kind === "url") {
```

Comment:
What about using [Buffer.isBuffer()](https://nodejs.org/api/buffer.html#buffer_class_method_buffer_isbuffer_obj) to determine which type of search it's using?